WordPress/Classes/ViewRelated/NUX/SignupUsernameTableViewController.swift

```diff
@@ -279,3 +281,23 @@ extension SignupUsernameTableViewController {
         }
     }
 }
+
+extension SearchTableViewCell {
```

Comment:
I wonder if we should add these methods to the `WordPressAuthenticator` lib. Wdyt?

Reply:
Yeah, totally agree. Should I do it?

Reply:
Yes, please! :)